<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmTcpClientClass</class>
 <widget class="QWidget" name="FrmTcpClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FrmTcpClient</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEditMain">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBoxData">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSend">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxHexReceive">
        <property name="text">
         <string>16进制接收</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxHexSend">
        <property name="text">
         <string>16进制发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxAscii">
        <property name="text">
         <string>控制字符</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxShow">
        <property name="text">
         <string>暂停显示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxDebug">
        <property name="text">
         <string>模拟设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxAutoSend">
        <property name="text">
         <string>定时发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxInterval">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>绑定地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxBindIP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>绑定端口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditBindPort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>服务器地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditServerIP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>服务器端口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditServerPort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonConnect">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClear">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonConnect</sender>
   <signal>clicked()</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_connected_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_save_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_clear_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSend</sender>
   <signal>clicked()</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_send_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxHexReceive</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>-21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxHexSend</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>-24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAscii</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>-32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxShow</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>-40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDebug</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAutoSend</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>-25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxInterval</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxBindIP</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditBindPort</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditServerIP</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditServerPort</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FrmTcpClientClass</receiver>
   <slot>on_checkbox_stateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>781</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_send_clicked()</slot>
  <slot>on_save_clicked()</slot>
  <slot>on_clear_clicked()</slot>
  <slot>on_connected_clicked()</slot>
  <slot>on_checkbox_stateChanged()</slot>
 </slots>
</ui>
